### PR TITLE
French support

### PR DESCRIPTION
**Pull request by TheRaphael0000**
Hello,

I have made a few changes to add the French support for the solver and the ui.

For the dictionary I use this repository : https://github.com/hbenbel/French-Dictionary/
I had to remove the "accents" because when we play Scrabble in French there are no accents on the letters.

I didn't implement the definition yet since I don't know any source for it.

There is also a strange behavior with the flag in the settings' menu.
Does it bother you to see what the problem is ? I have really no clue of  what it could be since I'm terrible in JS/Node.

Best.
Raphaël.

### CODE REVIEW

File: packages/scrabble-solver/package.json [hunk 1]
```diff
@@ -20,10 +20,10 @@
   },
   "homepage": "https://github.com/kamilmielnik/scrabble-solver#readme",
   "scripts": {
-    "build": "NEXT_TELEMETRY_DISABLED=1 next build",
```
Comment (TheRaphael0000) [inline packages/scrabble-solver/package.json:23]: Yeah I also had to remove this to be able to run it on my Windows computer. Sorry I forgot to mention it in the pull request message

Comment (kamilmielnik) [reply packages/scrabble-solver/package.json:23]: Windows build should be now fixed with https://github.com/kamilmielnik/scrabble-solver/pull/55, thanks for the report!

**Review by kamilmielnik — APPROVED** [review]
Thank you @TheRaphael0000, this is great :rocket: 

I made full French support a goal for 2.3.0: https://github.com/kamilmielnik/scrabble-solver/milestone/7

----

> I didn't implement the definition yet since I don't know any source for it.

I've created an issue for it: https://github.com/kamilmielnik/scrabble-solver/issues/52
I can do it with some basic crawling - can you maybe recommend an online french dictionary with definitions I could use for it?

----

> There is also a strange behavior with the flag in the settings' menu.
Does it bother you to see what the problem is ? I have really no clue of what it could be since I'm terrible in JS/Node.

No problem, I'll have a look at it and fix it after this PR is merged.
Can you just describe the issue a bit more or record a short video/screencast, please? :)

----

Please resolve conflicts and it's good to merge :fire:

**Comment by TheRaphael0000** [comment]
For the definition I did a little research, but I didn't find anything.

Here is the problem I'm having with the menu for the language.

image

**Comment by kamilmielnik** [comment]
> Here is the problem I'm having with the menu for the language.

Fixed with 69eb55f - thank you! :)

**Comment by kamilmielnik** [comment]
[2.3.0](https://github.com/kamilmielnik/scrabble-solver/releases/tag/2.3.0) with French has been released :tada:

**Pull request merged**